README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+The code should be a controller method because the controller determines what to do with the information and what view should be displayed back to the user.
```

Comment:
The answer we're actually looking for is a *helper* method. These are methods that just output HTML, or in some other way are concerned only with how things show up in a view.